### PR TITLE
[proxima-beam-core]: Fix name for reading batch source

### PR DESCRIPTION
**Pull request by LesTR**

### CODE REVIEW

File: beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java [hunk 1]
```diff
@@ -221,7 +221,7 @@ public void reload() {
             })
         .reduce((left, right) -> Union.of(left, right).output())
         .orElseThrow(failNotFound(attrs, "commit-log"))
-        .apply(filterAttrs(attrs));
+        .apply(String.format("get-stream-%s-filter-attrs-%s", name, attrs), filterAttrs(attrs));
```
Comment (je-ik) [inline beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java:224]: Can we remove this and the two following names for filter? That should be unnecessary. The rest can be merged. :+1:

File: beam/core/src/main/java/cz/o2/proxima/beam/direct/io/DirectDataAccessorWrapper.java [hunk 2]
```diff
@@ -99,6 +99,7 @@ public DirectDataAccessorWrapper(
 
     PCollection<StreamElement> ret =
         pipeline.apply(
+            "ReadBoundedBatch:" + uri,
```
Comment (je-ik) [inline beam/core/src/main/java/cz/o2/proxima/beam/direct/io/DirectDataAccessorWrapper.java:102]: This is probably the real fix. :+1:

File: beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java (hunk 1 above)
Comment (LesTR) [reply beam/core/src/main/java/cz/o2/proxima/beam/core/BeamDataOperator.java]: reverted 👍

**Comment by je-ik** [comment]
LGTM, we can merge after CI pass.

**Pull request merged**